mrjones2014/smart-splits.nvim
# feat(mux): Support splitting Kitty multiplexer panes

Fixes #205 

## How to test

Ensure you've run post-install hooks for kitty; I think you can do this via `:Lazy sync<cr>`

1. Use following config in a Kitty multiplexer session:

```lua
-- the actual keys `<M-C-k>` can be whatever you'd like
vim.keymap.set("n", "<M-C-k>", function()
    mux.split_pane("up")
end)
vim.keymap.set("n", "<M-C-j>", function()
    mux.split_pane("down")
end)
vim.keymap.set("n", "<M-C-h>", function()
    mux.split_pane("left")
end)
vim.keymap.set("n", "<M-C-l>", function()
    mux.split_pane("right")
end)
```

2. Use the keymaps, they should create kitty splits
